### PR TITLE
MDP-199: Popup update

### PR DESCRIPTION
**Pull request by OlehVynnyk**

### CODE REVIEW

File: src/components/Popup/InfoMessage.tsx [hunk 1]
```diff
@@ -5,5 +5,10 @@ interface IProps {
 }
 
 export const InfoMessage: FC<IProps> = ({ text }) => (
-  <div className="mt-20 mr-48 ml-24 font-lato text-16">{text}</div>
+  <div
+    style={{ lineHeight: "1.38" }}
```
Comment (pggl) [inline src/components/Popup/InfoMessage.tsx]: create custom tailwind class

File: src/components/Popup/Message.tsx [hunk 2]
```diff
@@ -5,5 +5,10 @@ interface IProps {
 }
 
 export const Message: FC<IProps> = ({ text }) => (
-  <div className="ml-24 mt-80 font-gilbert text-31">{text}</div>
+  <div
+    style={{ lineHeight: "1.1" }}
```
Comment (pggl) [inline src/components/Popup/Message.tsx]: create custom tailwind class

**Review by pggl — CHANGES_REQUESTED** [review]

**Comment by garySiladi** [comment]
could you pull from master to update PR?

File: src/components/Popup/InfoMessage.tsx (hunk 1 above)
Comment (pggl) [reply src/components/Popup/InfoMessage.tsx]: why this is not solved?

**Review by pggl — CHANGES_REQUESTED** [review]

**Review by slavomircesla — APPROVED** [review]

**Review by pggl — APPROVED** [review]

**Pull request merged**